apps/server/src/modules/provisioning/strategy/sanis/sanis-response.mapper.spec.ts

```diff
@@ -242,6 +257,28 @@ describe('SanisResponseMapper', () => {
 				expect(result?.[0].otherUsers).toBeUndefined();
 			});
 		});
+		describe('when no other participants are provided and FEATURE_OTHER_GROUPUSERS_PROVISIONING_ENABLED is true', () => {
+			const setup = () => {
+				Object.assign<IProvisioningFeatures, Partial<IProvisioningFeatures>>(provisioningFeatures, {
+					schulconnexOtherGroupusersEnabled: true,
+				});
+
+				const { sanisResponse } = setupSanisResponse();
+				sanisResponse.personenkontexte[0].gruppen![0].sonstige_gruppenzugehoerige = undefined;
+
+				return {
+					sanisResponse,
+				};
+			};
+
+			it('should set other users to undefined', () => {
```

Comment:
the description doesnt match the tests expectation. Should it be undefined or empty?